client/settings/settings-manager/duplicated-payment-methods-context.tsx

```diff
@@ -2,10 +2,11 @@
  * External dependencies
  */
 import { createContext } from 'react';
+import { PaymentMethodToPluginsMap } from '../../components/duplicate-notice';
 
 const DuplicatedPaymentMethodsContext = createContext( {
-	duplicates: [] as string[],
-	dismissedDuplicateNotices: [] as string[],
+	duplicates: {} as PaymentMethodToPluginsMap,
+	dismissedDuplicateNotices: {} as PaymentMethodToPluginsMap,
```

Comment:
This probably isn't a very big deal and definitely isn't breaking anything at the moment, but do you think we should maybe change the default value of this parameter that we supply to the provider over [here](https://github.com/Automattic/woocommerce-payments/blob/develop/client/settings/settings-manager/index.js#L211) to an empty object rather than an array as well?

For example...
```js
	const [
		dismissedDuplicateNotices,
		setDismissedDuplicateNotices,
	] = useState( wcpaySettings.dismissedDuplicateNotices || {} );
```
🤔

Reply:
Thanks, changed in 79084a2fd971c3771431ec32fbc1a4ec8984b3cc!